test/test_removedfiles.py

```diff
@@ -0,0 +1,286 @@
+#
+# Copyright © 2020 Red Hat, Inc.
+# Author(s): David Cantrell <dcantrell@redhat.com>
```

Comment:
You'll definitely want to update the copyright and author(s) section to reflect 2022 and you =)

Reply:
Adding that in the upcoming PR thanks